test/jest/Bladeburner/Actions.test.ts

```diff
@@ -61,85 +46,81 @@ describe("Bladeburner Actions", () => {
+        expect(bb.maxStamina).toBeGreaterThan(before);
       });
 
       it.each(<(keyof Skills)[]>["strength", "dexterity", "agility"])("awards %s exp", (stat: keyof Skills) => {
-        guaranteeSuccess(), (before = Player.exp[stat]), complete(Training);
+        (before = Player.exp[stat]), complete(train);
         expect(Player.exp[stat]).toBeGreaterThan(before);
       });
     });
 
     describe(BladeburnerGeneralActionName.HyperbolicRegen, () => {
-      const Regen = GeneralAction.createId(BladeburnerGeneralActionName.HyperbolicRegen);
+      const regen = GeneralAction.createId(BladeburnerGeneralActionName.HyperbolicRegen);
 
       it("heals the player", () => {
-        basicStats(), Player.takeDamage(Player.hp.max / 2), (before = Player.hp.current), complete(Regen);
+        Player.takeDamage(Player.hp.max / 2), (before = Player.hp.current), complete(regen);
         expect(Player.hp.current).toBeGreaterThan(before);
       });
 
       it("regains stamina", () => {
-        basicStats(), (inst.stamina = 0), complete(Regen);
-        expect(inst.stamina).toBeGreaterThan(0);
+        (bb.stamina = 0), complete(regen);
+        expect(bb.stamina).toBeGreaterThan(0);
       });
     });
 
     describe(BladeburnerGeneralActionName.Diplomacy, () => {
-      const Diplomacy = GeneralAction.createId(BladeburnerGeneralActionName.Diplomacy);
+      const diplomacy = GeneralAction.createId(BladeburnerGeneralActionName.Diplomacy);
 
       it("mildly reduces chaos in the current city", () => {
         let chaos;
-        basicStats(), allCitiesHighChaos(), ({ chaos } = inst.getCurrentCity()), complete(Diplomacy);
-        expect(inst.getCurrentCity().chaos).toBeGreaterThan(chaos * 0.9);
-        expect(inst.getCurrentCity().chaos).toBeLessThan(chaos);
+        allCitiesHighChaos(), ({ chaos } = bb.getCurrentCity()), complete(diplomacy);
+        expect(bb.getCurrentCity().chaos).toBeGreaterThan(chaos * 0.9);
+        expect(bb.getCurrentCity().chaos).toBeLessThan(chaos);
       });
 
       it("effect scales significantly with player charisma", () => {
-        basicStats(), Player.gainCharismaExp(1e500), allCitiesHighChaos(), complete(Diplomacy);
-        expect(inst.getCurrentCity().chaos).toBe(0);
+        Player.gainCharismaExp(1e500), allCitiesHighChaos(), complete(diplomacy);
+        expect(bb.getCurrentCity().chaos).toBe(0);
       });
 
       it("does NOT affect chaos in other cities", () => {
-        const otherCity = <CityName>CITIES.find((c) => c !== inst.getCurrentCity().name);
+        const otherCity = <CityName>cities.find((c) => c !== bb.getCurrentCity().name);
         /** Testing against a guaranteed 0-chaos level of charisma */
-        basicStats(), Player.gainCharismaExp(1e500), allCitiesHighChaos(), complete(Diplomacy);
-        expect(inst.cities[otherCity].chaos).toBeGreaterThan(0);
+        Player.gainCharismaExp(1e500), allCitiesHighChaos(), complete(diplomacy);
```

Comment:
I was apparently not looking closely enough... can you convert these to separate lines instead of using commas for sequencing?
Yes, it'll make the tests a lot longer (by linecount), but we don't really want to golf our tests.

Reply:
Yes, boss